packages/editor/src/client/locales/en.json

```diff
@@ -373,7 +373,10 @@
         "chooseImage": "Choose Image",
         "editImage": "Edit Image",
         "removeImage": "Remove Image",
-        "infoEditError": "Edit Token Error"
+        "infoEditError": "Edit Token Error",
+        "callToAction": "Call to action information",
+        "callToActionText": "Text",
+        "callToActionURL": "URL"
```

Comment:
Should translation keys be added into all the locales files? @0815pascal

Reply:
At the moment not.